VAGRANT.md

```diff
@@ -0,0 +1,33 @@
+# Vagrant support for pouch
+
+You can using Vagrant to quickly experience pouch or cross compile on non-linux.
+
+## Requirements
+
+* Vagrant 1.9.x or newer
+* VirtuaBox
+
+## Get started
+
+```bash
+vagrant up
+vagrant ssh -c "sudo su -l"
+
+# Start a nginx container with 80
+pouch run -d --name nginx -p 80:80 nginx
+curl http://localhost
+```
+
+## Build pouch with vagrant
+
+```bash
+export POUCH_BUILD=true # set POUCH_BUILD=true on Windows
```

Comment:
you means the `export xx=yy` in the ubuntu on window?

Reply:
On Windows export POUCH_BUILD=true doesn't,